src/Sql/Sql/help/New-AzSqlDatabaseImport.md

```diff
@@ -14,11 +14,13 @@ Imports a .bacpac file and create a new database on the server.
 ## SYNTAX
 
 ```
-New-AzSqlDatabaseImport -DatabaseName <String> -Edition <DatabaseEdition> -ServiceObjectiveName <String>
- -DatabaseMaxSizeBytes <Int64> [-ServerName] <String> -StorageKeyType <StorageKeyType> -StorageKey <String>
+New-AzSqlDatabaseImport -Edition <DatabaseEdition> -ServiceObjectiveName <String> -DatabaseMaxSizeBytes <Int64>
```

Comment:
Please double check if this change is correct. I suspect it makes breaking change.

Reply:
Reverted the change to positional parameters